vars/create_windows_template_vars.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# The variables in this file are copied from file "test.yml" for creating Windows
+# OVF template testing. It's used with "windows/create_template_playbook_list.yml".
+# The commented out variables are optional.
+#
+exit_testing_when_fail: True
+
+# Set 'proxy:port' when downloading files from external servers requiring proxy on
+# your local machine.
+# http_proxy_localhost: ""
+
+# vCenter Server information is optional
+# vcenter_hostname: "xx.xx.xx.xx"
+# vcenter_username: "xxxxxx@xx.xx"
+# vcenter_password: "xxxxxx"
+# datacenter: "xxxxxx"
+
+# ESXi Server information is mandatory
+esxi_hostname: "xx.xx.xx.xx"
+esxi_username: "root"
+esxi_password: "xxxxxx"
+
+datastore: "datastore1"
+# validate_certs: False
+
+vm_name: "windows_template_ansible"
+vm_username: "test"
+vm_password: "B1gd3m0z"
+# vm_folder: ''
+new_vm: True
+hardware_version: 16
+
+# Guest ID:
+# Windows 10 32bit: Windows9Guest,
+# Windows 10 64bit: Windows9_64Guest,
+# Windows 11 64bit: Windows11_64Guest,
+# Windows Server 2016: Windows9Server64Guest,
+# Windows Server 2019: Windows2019srv_64Guest,
+# Windows Server 2022: Windows2019srvNext_64Guest
+#
+guest_id: "Windows9_64Guest"
+
+# VM Configuration:
+memory_mb: 4096
+cpu_number: 2
+cpu_cores_per_socket: 1
+firmware: "efi"
+secureboot_enabled: True
+boot_disk_controller: "paravirtual"
+boot_disk_size_gb: 64
+network_adapter_type: "e1000e"
+# vm_network_name: "VM Network"
+
+os_installation_iso_list:
+  - "[datastore1] en-us_windows_10_business_editions_version_21h2_x64_dvd_ce067768.iso"
+unattend_install_conf: "Windows/win10/64bit/efi/Autounattend.xml"
+# windows_product_key: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
+
+# Select one of the below VMware tools sources:
+# (1) ESXi host bundled VMware tools
+# vmtools_esxi_bundled: True
+
+# (2) VMware tools download URL
+vmtools_url_path: "https://packages.vmware.com/tools/releases/latest/windows/VMware-tools-windows-11.3.5-18557794.iso"
```

Comment:
I suggest replacing "latest" with "11.3.5", otherwise when newer tools version is released, this URL will become invalid.